vk-video/examples/player/player/shader.wgsl

```diff
@@ -0,0 +1,46 @@
+struct VertexInput {
+    @location(0) position: vec3<f32>,
+    @location(1) tex_coords: vec2<f32>,
+}
+
+struct VertexOutput {
+    @builtin(position) position: vec4<f32>,
+    @location(0) tex_coords: vec2<f32>,
+}
+
+@vertex
+fn vs_main(input: VertexInput) -> VertexOutput {
+    var output: VertexOutput;
+
+    output.position = vec4(input.position, 1.0);
+    output.tex_coords = input.tex_coords;
+
+    return output;
+}
+
+@group(0) @binding(0) var y_texture: texture_2d<f32>;
+@group(0) @binding(1) var uv_texture: texture_2d<f32>;
+@group(0) @binding(2) var sampler_: sampler;
+
+fn srgb_to_linear(srgb: vec3<f32>) -> vec3<f32> {
```

Comment:
This is only needed if you write to SRGB texture. I added that in the shader because WASM does not support multiple TextureViews on single Texture, but a better way would be to remove this conversion and when writing to Srgb texture use non-Srgb view

Reply:
oh, ok, I'll try that